devops/vaults.env

```diff
@@ -1,5 +1,5 @@
-# Base Path Openshift: businesses/edit Firebase:
-VUE_APP_PATH=
+# Base Path Openshift: /businesses/edit Firebase: /
+VUE_APP_PATH="/"
```

Comment:
Did you run this app locally to test this?

Reply:
Yes, test local and in firebase.

Reply:
Excellent - thanks!